src/components/equipment/BarcodeScanner.tsx

```
@@ -0,0 +1,178 @@
+            {/* Barcode input */}
+            <div className="mb-4">
+                <Label htmlFor="barcode">Barcode</Label>
+                <div className="flex gap-2">
+                    <Input
+                        id="barcode"
+                        value={barcode}
+                        onChange={(e) => setBarcode(e.target.value)}
+                        placeholder="Scan or type barcode..."
+                        className="font-mono"
+                    />
+                    <Button variant="outline" onClick={() => setBarcode("DEMO-001")}>
```

ESLint

[error] Delete `··········`

(prettier/prettier)